.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -181,14 +182,15 @@ jobs:
         THREADS_LIMIT: 20
         RUN_TYPE: 'perf_ci'
         ENABLE_PROMETHEUS_SNAPSHOT: 'True'
-        WINDOWS_URL: ${{ secrets.PERF_WINDOWS_URL }}
+        WINDOWS_SERVER_2019_URL: ${{ secrets.PERF_WINDOWS_SERVER_2019_URL }}
+        WINDOWS11_URL: ${{ secrets.PERF_WINDOWS11_URL }}
       run: |
         build=$(pip freeze | grep benchmark-runner | sed 's/==/=/g')
         build_version="$(cut -d'=' -f2 <<<"$build")"
         echo '>>>>>>>>>>>>>>>>>>>>>>>>>> Start E2E workload: ${{ matrix.workload }} >>>>>>>>>>>>>>>>>>>>>>>>>>'
         scp -r "$RUNNER_PATH/.kube/config" provision:"$CONTAINER_KUBECONFIG_PATH"
         # SCALE: Run vdbench/ bootstorm
-        if [[ '${{ matrix.workload }}' == 'vdbench_pod_scale' || '${{ matrix.workload }}' == 'vdbench_kata_scale' || '${{ matrix.workload }}' == 'vdbench_vm_scale' || '${{ matrix.workload }}' == 'bootstorm_vm_scale' || '${{ matrix.workload }}' == 'windows_vm_scale' ]]
+        if [[ '${{ matrix.workload }}' == 'vdbench_pod_scale' || '${{ matrix.workload }}' == 'vdbench_kata_scale' || '${{ matrix.workload }}' == 'vdbench_vm_scale' || '${{ matrix.workload }}' == 'bootstorm_vm_scale' || '${{ matrix.workload }}' == 'windows_vm_windows_server_2019_scale' || '${{ matrix.workload }}' == 'windows_vm_windows11_scale' ]]
```

Comment:
It would be good to clean this up some time so we don't keep just adding more cases, although it might be a bit awkward...

Reply:
@RobertKrawitz, You right fixed it.